client/src/components/related/Related.jsx

```diff
@@ -10,53 +10,50 @@ const Related = () => {
   //**************
   let [relatedProductsId, setProductsId] = useState([]);
   let [relatedProductsDetail, setDetail] = useState([]);
-  let [relatedProductsStyles, setStyles] = useState([]);
 
   // *************
   // Initial Renders of Data
   // *************
   useEffect(() => {
-    // First axios GET for related products ID
     axios.get('/related', {params: {
-      product_id: 63609
-    }
-  })
-  .then((res) => {
-    setProductsId(relatedProductsId = res.data);
-    // Second axios GET for related products info using ID
-    axios.get('/related/products', {params: {
-      product_id: relatedProductsId
-        }
-      })
-      .then((details) => {
-        setDetail(relatedProductsDetail = details.data);
-        // Third axios GET for styling data using ID
-        axios.get('/related/products/styles', {params: {
+        product_id: 63609
```

Comment:
Eventually will probably want to get the `product_id` from the App state

Reply:
Yea, I will try to do that in the next branch